tools-local/tasks/mobile.tasks/AndroidAppBuilder/Templates/MonoRunner.java

```diff
@@ -28,6 +28,8 @@
 public class MonoRunner extends Instrumentation
 {
     static {
+        // loadLibrary triggers JNI_OnLoad in these libs
+        System.loadLibrary("System.Security.Cryptography.Native.OpenSsl");
```

Comment:
should we follow `System.Security.Cryptography.Native.Apple` and use a separate`System.Security.Cryptography.Native.Android` folder/library?

Reply:
yep, that's the plan 🙂

Reply:
…if the plan is to have a separate `System.Security.Cryptography.Native.Android` folder, then we *certainly* should use an `Android` prefix on the C exports…